LibTest/io/HttpClient/findProxy_A01_t01.dart

```diff
@@ -67,18 +64,21 @@ test() async {
 
   client.authenticateProxy =
       (String host, int port, String scheme, String? realm) {
-        authenticateProxyCalled = true;
-        Completer<bool> completer = new Completer<bool>();
-        completer.complete(true);
-        return completer.future;
+    authenticateProxyCalled = true;
+    Completer<bool> completer = new Completer<bool>();
```

Comment:
can this be simplified to `return Future<bool>.value(true);` instead?
Or maybe you can just do `Expect.fail("authenticaterProxy should not be called");`

Reply:
@aam thank you. This makes sense. Code simplified